docs/DeveloperGuide.adoc

```diff
@@ -98,9 +98,9 @@ image::LogicClassDiagram.png[]
 *API* :
```

Comment:
For Figure 3. Component interactions for delete 1 command, the arrow seems to overlap with the activation bar. 
<img width="234" alt="Screenshot 2019-10-25 at 11 37 09 AM" src="https://user-images.githubusercontent.com/43085776/67541649-56dd4f00-f71c-11e9-9f75-a851452be826.png">

For the self-invocation part, it might be better if the line in the middle can be covered.
<img width="191" alt="Screenshot 2019-10-25 at 11 40 03 AM" src="https://user-images.githubusercontent.com/43085776/67541756-b3406e80-f71c-11e9-980f-a7a27a7dbc6a.png">